glossary.md

```diff
@@ -5,3 +5,66 @@ css_id: glossary
 
 # Glossary
 
+### Conformance terminology
+The keywords MUST, MUST NOT, REQUIRED, SHALL, SHALL NOT, SHOULD, SHOULD NOT, RECOMMENDED, MAY, and OPTIONAL in this document are to be interpreted as described in {{RFC2119}}.
+
+In order to be considered Uptane-compliant, an implementation MUST follow all of these rules as specified in the document.
```

Comment:
This line doesn't really make sense here.

Reply:
Good catch. I'll remove it.
```suggestion

```

Reply:
I'm going to re-open a pull request here to address your comments.